generic/tests/cfg/kdump.cfg

```diff
@@ -16,6 +16,20 @@
     # crash_cmd = "echo c > /proc/sysrq-trigger"
     # If you want configure multiline, split lines with ";", like:
     # kdump_config = configure line1; config line2
+    # "crashkernel=" pattern changed after RHEL.9
+    # Setting the crashkernel default parameters based on different arches
+    # Pls modify the params if needed
+    x86_64:
+        crashkernel_default_params = 1G-4G:192M,4G-64G:256M,64G-:512M
+    aarch64:
+        crashkernel_default_params = 1G-4G:256M,4G-64G:320M,64G-:576M
+    ppc64le, ppc64:
+        crashkernel_default_params = 2G-4G:384M,4G-16G:512M,16G-64G:1G,64G-128G:2G,128G-:4G
```

Comment:
On ppc64le:
Only RHEL9.x guest is supported on RHEL8 host. Can you help to move this to RHEL.8 part and I think we need to distinguish  guest RHEL 8 guest betwween RHEL 9 guest there too.